econml/bootstrap.py

```diff
@@ -35,16 +35,25 @@ class BootstrapEstimator(object):
 
     n_jobs: int, default: None
         The maximum number of concurrently running jobs, as in joblib.Parallel.
+
+    compute_means : bool, default: True
+        Whether to pass calls through to the underlying collection and return the mean
+
+    prefer_wrapped: bool, default: False
```

Comment:
Same for this. An example of where we can use this would help. These two arguments are bit cryptic.